apps/balancer-tools/src/app/apr/round/(components)/ClientSideBodyRow.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+import { networkFor } from "@bleu-balancer-tools/utils";
+import cn from "clsx";
+import { useRouter } from "next/navigation";
+import React from "react";
+
+export default function ClientSideBodyRow({
+  children,
+  classNames,
+  poolNetwork,
+  poolId,
+  roundId,
+}: React.PropsWithChildren<{
+  classNames?: string;
+  poolNetwork: number;
+  poolId: string;
+  roundId: string;
+}>) {
+  const { push } = useRouter();
+  function redirectToPool(network: number, poolID: string, roundID: string) {
+    const poolRedirectURL = roundID
+      ? `/apr/pool/${networkFor(network)}/${poolID}/round/${roundID}`
+      : `/apr/pool/${networkFor(network)}/${poolID}`;
+    push(poolRedirectURL);
+  }
```

Comment:
❌ just use a link

Reply:
For now, I'm using the same approach [as Internal Manager](https://github.com/bleu-studio/balancer-tools/pull/152#discussion_r1258458339). But I will refactor this to use tailwind table classes and therefore Link